plugins/docker/docker_

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""
+=head1 NAME
+
+docker_ - Docker wildcard-plugin to monitor a L<Docker|https://www.docker.com> host.
+
+This wildcard plugin provides at the moment only the suffixes C<containers>, C<images>, C<status>,
```

Comment:
Just for the inspiration: maybe it would be nice to switch this plugin from the wildcard style to a multigraph plugin? In this case the plugin would output the configuration (or values) of all sections and just add `multigraph docker_cpu` (and so on) lines in front of every block.

This would simplify the plugin installation (multiple symlinks are not required). But if it is common to use only a few of these plugins, then a wildcard plugin could be more suitable.

Reply:
I never used multigraph plugins but it can be a good thing. I will take a look on this type of plugin.